spellbook/models/transfers/ethereum/erc1155/transfers_ethereum_erc1155_rolling_day.sql

```diff
@@ -0,0 +1,18 @@
+{{ config(
+        alias ='erc1155_rolling_day',
+        materialized ='view'
```

Comment:
is this redundant? is view the default materialization for this directory in the dbt project?

Reply:
Yes that's right ! Just modified this